src/server/database/Database/Implementation/CharacterDatabase.cpp

```diff
@@ -57,6 +57,8 @@ void CharacterDatabaseConnection::DoPrepareStatements()
     PrepareStatement(CHAR_SEL_CHAR_LEVEL, "SELECT level FROM characters WHERE guid = ?", CONNECTION_SYNCH);
     PrepareStatement(CHAR_SEL_CHAR_ZONE, "SELECT zone FROM characters WHERE guid = ?", CONNECTION_SYNCH);
     PrepareStatement(CHAR_SEL_CHARACTER_NAME_DATA, "SELECT race, class, gender, level FROM characters WHERE guid = ?", CONNECTION_SYNCH);
+    PrepareStatement(CHAR_SEL_CHARACTER_DATA_NAME, "SELECT race, class, gender, `name` FROM characters WHERE guid = ?", CONNECTION_SYNCH);
```

Comment:
its is my use ..... and some times its better to use ` ` in the query ... but i forget to remove this here , sorry